sdk/loadtestservice/arm-loadtestservice/test/loadTestResourceOperations.ts

```diff
@@ -0,0 +1,208 @@
+/*
```

Comment:
@venkatr21 Thanks for your effort on contributing the test cases. The overall looks good to me. Could you change the test file name to `loadTestResourceOperations.spec.ts`?

Reply:
@MaryGao Sorry for the delay in response.
I have created a new PR for the same purpose https://github.com/Azure/azure-sdk-for-js/pull/24272
I am abandoning this PR, since it is a little outdated